### ISSUE TITLE
Add CI service for builds (incl link checking)

### ISSUE THREAD

**Issue by dcwalk** [ixmaps/website2017#27]
I'd be up for adding CI to check builds and whether links are working, I've used Travis before, but not for php projects. Maybe we could write a test for links?

- https://docs.travis-ci.com/user/for-beginners
- https://github.com/travis-ci-examples/php
- https://stackoverflow.com/questions/15770903/check-if-links-are-broken-in-php

**Comment by colinmccann** [comment]
Yes plz. I've never used one before, so defer to your judgement - Travis good?
Love this idea - start small with eg link testing (inc https checks maybe?).
Do you want to integrate? If so, could you provide a small primer (email or README?) at some point. Thanks DW!

**Comment by dcwalk** [comment]
Oh, just realized I framed this wrong, we'll want to test the build process too. I'm super swamped but will circle back on this early next week.

**Comment by Andrew-Clement** [comment]
Is this still an issue, or can it be closed?

**Comment by dcwalk** [comment]
Nothing has been added, this should stay open
